cookiecutter.json

```diff
@@ -8,11 +8,6 @@
   "author_name": "CERN",
   "author_email": "info@{{ cookiecutter.project_site }}",
   "year": "{% now 'local', '%Y' %}",
-  "python_version": [
-    "3.8",
```

Comment:
moderate: we could just leave 3.9 as an option. Leaving the flexibility to add new versions in the future if possible/desired. That way we also avoid changing the pipfile since it will be autofilled.

```python
"python_version": [
  "3.9",
 ]
 ```

Reply:
I am not sure what is the advantage, given that there will be always one possible version...